content/en/docs/tasks/debug/debug-cluster/_index.md

```diff
@@ -314,6 +316,13 @@ This is an incomplete list of things that could go wrong, and how to adjust your
   - Mitigates: Node shutdown
   - Mitigates: Kubelet software fault
 
+### Retaining old settings after cluster upgrade
```

Comment:
```suggestion
### Things to note after dockershim migration
```

Reply:
Alright , I'll push the changes as required.